layers/test/test_two_layer_net.py

```diff
@@ -0,0 +1,127 @@
+import unittest
+import numpy as np
+
+from classifiers.softmax import ClassifierSoftmax
+from layers.relu import LayerReLU
+from layers.fully_connected import LayerFullyConnected
+from regularizers.l2 import RegularizerL2
+
+from lib.gradient_check import eval_numerical_gradient
+from utils.compose import compose
+
+
+class TestTwoLayerNetFixture(unittest.TestCase):
+    def initConstants(self):
+        pass
```

Comment:
Might as well raise the new [NotImplementedError](https://docs.python.org/3/library/exceptions.html#NotImplementedError) here